README.md

```diff
@@ -45,31 +45,38 @@ Tart Executor is open sourced under MIT license so people can base their own exe
 Tart itself on the other hand is [source available under Fair Software License](https://tart.run/licensing/)
 that required paid sponsorship upon exceeding a free limit.
 
-## Advanced configuration
+## Supported command-line arguments
 
-By default, for preparing a virtual machine and running scripts inside of it Tart Executor assumes that
-your virtual machines can be SSH-ed into `admin` user via `admin` password.
+### `prepare` stage
 
-```toml
-[runners.custom]
-  prepare_exec = "gitlab-tart-executor"
-  prepare_args = ["prepare", "--username=gitlab", "--password=password101"]
-  run_exec = "gitlab-tart-executor"
-  run_args = ["run", "--username=gitlab", "--password=password101"]
-  cleanup_exec = "gitlab-tart-executor"
-  cleanup_args = ["cleanup"]
-```
+| Argument  | Default           | Description                                                     |
+|-----------|-------------------|-----------------------------------------------------------------|
+| `--cpu`   | `0` (no override) | Override default image CPU configuration (number of CPUs)       |
+| `--memory` | `0` (no override) | Override default image memory configuration (size in megabytes) |
 
-`prepare` command has arguments to override CPU/Memory of cloned virtual machines.
-For a full list of options please refer to `gitlab-tart-executor prepare --help`.
+## Supported environment variables
+
+| Name                      | Default | Description                                                                                                                          |
+|---------------------------|---------|--------------------------------------------------------------------------------------------------------------------------------------|
+| `TART_EXECUTOR_SSH_USERNAME` | admin   | SSH username to use when connecting to the VM                                                                                        |
+| `TART_EXECUTOR_SSH_PASSWORD` | admin   | SSH password to use when connecting to the VM                                                                                        |
+| `TART_EXECUTOR_HEADLESS`     | true    | Run the VM in headless mode (`true`) or with GUI (`false`)                                                                           |
+| `TART_EXECUTOR_ALWAYS_PULL`  | true    | Always pull the latest version of the Tart image (`true`) or only when the image doesn't exist locally (`false`)                     |
+| `TART_EXECUTOR_SOFTNET`      | false   | Whether to enable [Softnet](https://github.com/cirruslabs/softnet) software networking (`true`) or disable it (`false`)              |
+| `TART_EXECUTOR_HOST_DIR`     | false   | Whether to mount a temporary directory from the host for performance reasons (`true`) or use a directory inside of a guest (`false`) |
```

Comment:
Let's addd a note that both host and guest should be Ventura+.

Reply:
Added in https://github.com/cirruslabs/gitlab-tart-executor/pull/11/commits/543ba29ed00cb20c3560391405e1572bbf3bbe38.